jekyll/_cci2/resource-class-overview.adoc

```diff
@@ -0,0 +1,121 @@
+
+[source,yaml]
+----
+jobs:
+  my-job:
+    docker: # use the Docker executor
+      - image: cimg/base:2022.09 # specify a Docker image
+    resource_class: xlarge # specify a resource class
+    steps:
+      - run: echo "Hello World"
+----
+
+Executor resource classes are specified by size, for example, `small` or `medium`. Each size corresponds to specific CPU and RAM requirements. For a full list of available resource class options, see the link:/docs/configuration-reference/#resourceclass[configuration reference]. The link:https://circleci.com/product/features/resource-classes[resource class product page] provides information on credit usage per resource class, and resource class availability for each plan type.
+
+When using CircleCI's self-hosted runners, use the `resource_class` key in your project configuration to specify which runner to use for a job. Self-hosted runner resource classes are assigned and configured during the link:/docs/runner-installation/[installation process].
+
+[source,yaml]
+----
+jobs:
+  my-job:
+    machine: true
+    resource_class: <my-namespace>/<my-runner>
+    steps:
+      - run: echo "Hello runner"
+----
+
+Another use of the `resource_class` key is to specify an execution environment. For example, link:/docs/using-arm[Arm] is a resource class option when using the `machine` executor.
+
+[source,yaml]
+----
+jobs:
+  my-job:
+    machine:
+      image: ubuntu-2004:202101-01
+    resource_class: arm.medium
+    steps:
+      - run: uname -a
+      - run: echo "Hello, Arm!"
+----
+
+[#quickstart]
+== Quickstart
+
+For config examples showing how to use the `resource_class` key to **specify CPU and RAM size** for an execution environment, see the following sections of the configuration reference page:
+
+* link:/docs/configuration-reference/#docker-execution-environment[Docker resource classes]
+* link:/docs/configuration-reference/#linuxvm-execution-environment[Linux VM resource classes]
+* link:/docs/configuration-reference/#macos-execution-environment[macOS resource classes]
+* link:/docs/configuration-reference/#windows-execution-environment[Windows resource classes]
+* GPU resource classes: link:/docs/configuration-reference/#gpu-execution-environment-linux[Linux] and link:/docs/configuration-reference/#gpu-execution-environment-windows[Windows]
+* link:/docs/configuration-reference/#arm-execution-environment-linux[Arm resource classes]
+
+To get started setting up a **self-hosted runner**, see the link:/docs/runner-installation[Installing self-hosted runners with the web app] page.
+
+[#default-resource-class]
+== Default resource class
+
+If a `resource_class` is not explicitly declared for a job, CircleCI will use a default resource class size for your organization. Defaults are subject to change. It is best practice to specify a resource class, rather than relying on a default.
```

Comment:
```suggestion
If a `resource_class` is not explicitly declared for a job, CircleCI will use a default resource class size. Defaults are subject to change. It is best practice to specify a resource class, rather than relying on a default.
```